Devscord.DiscordFramework/Framework/Server.cs

```diff
@@ -58,5 +62,15 @@ public static IMongoDatabase GetDatabase()
         {
             return _database;
         }
+
+        private static Task _client_UserJoined(SocketGuildUser guildUser)
+        {
+            var channelContext = (new ChannelContextFactory()).Create(guildUser.Guild.DefaultChannel);
+            var userContext = (new UserContextsFactory()).Create(guildUser);
+            var discordServerContext = (new DiscordServerContextFactory()).Create(guildUser.Guild);
+
+            var userService = new UserService();
+            return userService.WelcomeUser(channelContext, userContext, discordServerContext);
```

Comment:
myśle że to powinno przepływać przez workflow i jakieś nietypowe middleware

ewentualnie to powinna być komenda po przejściu na CQRS

druga opcja wydaje się prostsza
planuje niedługo wrzucić CQRS i wtedy się to poprawi
żeby nie robić poprawek na kodzie który i tak zmienimy, a jest do ogarnięcia